.gitignore

```diff
@@ -62,3 +63,7 @@ prediction/data/*
 
 # ignore data
 data/
+
+
+# ignore Sphinx apidoc
+_apidoc_*
```

Comment:
Please revert this change - see #80 (in particular https://github.com/concept-to-clinic/concept-to-clinic/pull/80/files)